layers/+lang/ipython-notebook/packages.el

```diff
@@ -68,7 +68,9 @@
         "+" 'ein:notebook-worksheet-insert-next
         "-" 'ein:notebook-worksheet-delete
         "x" 'ein:notebook-close
-        "u" 'ein:worksheet-change-cell-type
+        "u" 'ein:worksheet-change-cell-type)
+
+      (evil-leader/set-key-for-mode 'ein:notebook-multilang-mode
```

Comment:
shouldn't it be `spacemacs/set-leader-keys-for-major-mode`? (the same as the key-bindings above)